incubator-tools/dialogflowcx_email_ingestion/dialogflowcx_email_ingestion.ipynb

```diff
@@ -0,0 +1,1098 @@
+{
```

Comment:
<p>This notebook doesn't seem to use Document AI at all. It uses AutoML Entity Extraction and Dialogflow. I think this would be better suited for <a href="https://github.com/GoogleCloudPlatform/contact-center-ai-samples" target="_blank">https://github.com/GoogleCloudPlatform/contact-center-ai-samples</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Removing the tool from here and we'll create a PR to update in the specified directory.